pages/transcripts/christoph-nakazawa.md

```diff
@@ -0,0 +1,195 @@
+---
+title: Christoph Nakazawa
+date: Thursday July 13th
+---
+## Engineer at Facebook London · Jest · Yarn · RNP
+
+**Christoph Nakazwa:**  Just to introduce myself, I'm Christoph and I manage the JavaScript Tools team at Facebook London. Currently we work on Yarn, Jest and Metro Bundler (https://github.com/facebook/metro-bundler) which powers React Native.
```

Comment:
I think what I said here about Metro Bundler and RN might have been a bit strong. How about: "Just to introduce myself, I'm Christoph and I manage the JavaScript Tools team at Facebook London. Currently we work on Yarn, Jest and [Metro Bundler](https://github.com/facebook/metro-bundler) which provides the development server and bundling infrastructure for React Native."